Skin/FrameXML/CharacterFrame.lua

```diff
@@ -58,6 +58,8 @@ function private.FrameXML.CharacterFrame()
     ClassBackground:SetAlpha(0.4)
 
 
+    CharacterStatsPane.ItemLevelCategory.Title:Hide()
+    CharacterStatsPane.ItemLevelCategory.Background:Hide()
```

Comment:
What is your goal here? The `ItemLevelCategory` frame gets hidden [aurora.lua line 147](https://github.com/Haleth/Aurora/blob/master/aurora.lua#L147)

Additionally, since changing Aurora to being an [embeddable addon](https://github.com/Haleth/Aurora/issues/81), I prefer to keep the main layout of the skin pretty close to Blizzard. Any significant differences should be in the OnLoad handler in aurora.lua

Reply:
Sorry, my mistake, my problem caused by other addon.